app/controllers/bulk_actions_controller.rb

```diff
@@ -57,7 +57,10 @@ def bulk_action_params
       :pids,
       manage_release: %i[tag what who to],
       set_governing_apo: [:new_apo_id],
-      manage_catkeys: [:catkeys],
+      set_catkeys_and_barcodes: %i[
+        catkeys catkeys_option
+        barcodes barcodes_options
```

Comment:
Should `barcodes_options` be `barcodes_option` for consistency w/ catkeys, licenses, and such?